packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -30,8 +29,6 @@ export type RefreshSummaryResult =
 	| {
 			latestSummaryUpdated: true;
 			wasSummaryTracked: false;
-			snapshotTree: ISnapshotTree;
```

Comment:
Please update the comment for case 3 above

Reply:
Updated